week_3/week_3_quiz.md

```diff
@@ -0,0 +1,130 @@
+# Week 3 Quiz
+
+### Quiz Instructions
+
+In this Week 3 Quiz, we will ask you a few questions about ABCD (Sampling, Recruitment, Retention) and you will complete a few ReproNim exercises (Containers and ReproEnv).
+
+***
+
+### ABCD (Sampling, Recruitment, Retention) Questions
+
+First, we will ask you a few questions about ABCD Data.
+
+**Question 1**
+
+The ABCD sample is representative of youth in the United States.
+
+- True
+- False
+
+**Question 2**
+
+What was the primary method of recruitment for single birth children?
+
+- Birth registries
+- Mailing lists
+- Schools
+- Summer camps
+
+**Question 3**
+
+What was the primary method of recruitment for twins?
+
+- Birth registries
+- Mailing lists
+- Schools
+- Summer camps
+
+**Question 4**
+
+What are two sources of sample bias the ABCD recruitment strategy could not account for due to constraints of the study?
+
+- Urbanicity & Self-Selection
+- Race & Sex
+- Socio-Economic Status & Race
+- Urbanicity & Sex
+- Self-Selection & Socio-Economic Status
+
+**Question 5**
+
+How has the sociodemographic bias in missing assessments changed after the beginning of the COVID pandemic?
+
+- Increased
+- Stayed the same
+- Decreased
+
+### ReproNim (Containers) Questions
+
+Some of these questions are adapted, with modification, from the [Software Carpentries](http://software-carpentry.org) lesson on [Reproducible Computational Environments using Containers](https://carpentries-incubator.github.io/docker-introduction/index.html), which is licensed under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode). Other questions were adapted, with modification, from the [Neurohackweek](http://neurohackweek.github.io/) lesson on [Docker for Scientists](https://neurohackweek.github.io/docker-for-scientists/), which is licensed under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode).
+
+The following questions require Docker to be installed on your computer. Please [download Docker](https://www.docker.com/products/docker-desktop) and then follow the installation instructions.
+
+**Question 6**
+
+Once Docker is installed, try downloading and running the alpine Docker container. Which of the following terminal commands would download and run the alpine Docker container? (Select all that apply)
+
+- [ ]  `docker run alpine`
```

Comment:
Do you want the checkboxes for all the questions? I'm fine with or without. Or does this specifically represent the possibility you can select multiple answers? (in which case a question in [quiz_2 should be changed](https://github.com/ABCD-ReproNim/exercises/blame/6b7f012ae33b006346b3230f75cc28b397edfd84/week_2/week_2_quiz.md#L21), but not the responsibility of this pull request I think)

Reply:
Ahh, I spend like 10 seconds looking for how to do the square bullet but didn't find it. I like it and I can change week_2 to match

Reply:
I changed the bullet points to radio buttons using html and kept the checkboxes for ones where multiple answers can be selected at once.